Batteries/Data/Vector/Lemmas.lean

```diff
@@ -43,23 +43,25 @@ theorem toArray_injective : ∀ {v w : Vector α n}, v.toArray = w.toArray → v
 
 /-! ### tail lemmas -/
 
-theorem tail_zero {v : Vector α 0} : v.tail = #v[] := Vector.eq_empty
+theorem tail_eq_of_zero {v : Vector α 0} : v.tail = #v[] := Vector.eq_empty
 
-theorem tail_ne_zero [NeZero n] {v : Vector α n} :
+theorem tail_eq_of_ne_zero [NeZero n] {v : Vector α n} :
     v.tail = v.eraseIdx 0 n.pos_of_neZero := dif_pos _
 
 @[simp] theorem toList_tail {v : Vector α n} :
     v.tail.toList = v.toList.tail :=
   match n with
-  | 0 => by simp only [tail_zero, Vector.eq_empty, List.tail_nil]
-  | _ + 1 => by simp only [tail_ne_zero, Vector.toList_eraseIdx, List.eraseIdx_zero]
+  | 0 => by simp only [tail_eq_of_zero, Vector.eq_empty, List.tail_nil]
+  | _ + 1 => by simp only [tail_eq_of_ne_zero, Vector.toList_eraseIdx, List.eraseIdx_zero]
```

Comment:
Please change these to ordinary simps since they are terminal.

Reply:
Pushed a change - is that what you mean?

Reply:
I mean they should be `simp` and not `simp only`.

Reply:
Right - I did that and then omitted the lemmas which are already in the simp set.